core/services/im_here/MavlinkNMEA.py

```diff
@@ -0,0 +1,86 @@
+from enum import IntEnum
+from typing import Optional
+
+import pynmea2
+from pydantic import BaseModel
+
+from exceptions import UnsupportedSentenceType
+
+
+class GPS_FIX_TYPE(IntEnum):
+    GPS_FIX_TYPE_NO_GPS = 0
+    GPS_FIX_TYPE_NO_FIX = 1
+    GPS_FIX_TYPE_2D_FIX = 2
+    GPS_FIX_TYPE_3D_FIX = 3
+    GPS_FIX_TYPE_DGPS = 4
+    GPS_FIX_TYPE_RTK_FLOAT = 5
+    GPS_FIX_TYPE_RTK_FIXED = 6
+    GPS_FIX_TYPE_STATIC = 7
+    GPS_FIX_TYPE_PPP = 8
+
+
+class GPS_INPUT_IGNORE_FLAG(IntEnum):
```

Comment:
Since this is a flag, it should use enum.IntFlag

Reply:
Didn't know this exists! Will use.